cartography/intel/gcp/compute.py

```diff
@@ -147,6 +227,171 @@ def load_gcp_instances(neo4j_session, data, gcp_update_tag):
             Hostname=instance.get('hostname', None),
             gcp_update_tag=gcp_update_tag
         )
+        _attach_gcp_nics(neo4j_session, instance, gcp_update_tag)
+
+
+def load_gcp_vpcs(neo4j_session, vpcs, gcp_update_tag):
+    """
+    Ingest VPCs to Neo4j
+    :param neo4j_session: The Neo4j session object
+    :param vpcs: List of VPCs to ingest
+    :param gcp_update_tag: The timestamp value to set our new Neo4j nodes with
+    :return: Nothing
+    """
+    query = """
+    MATCH(p:GCPProject{id:{ProjectId}})
+    MERGE(vpc:GCPVpc{id:{PartialUri}})
```

Comment:
[question] what guaranty this to be unique across GCP?

Reply:
ProjectIDs are unique across GCP : 

"The project number and project ID are unique across Google Cloud Platform. If another user owns a project ID for their project, you won't be able to use the same project ID." -- https://cloud.google.com/resource-manager/docs/creating-managing-projects